### ISSUE TITLE
Breacks backspace functionality in windows

### ISSUE THREAD

**Issue by GoogleCodeExporter** [noctarius/chromebackspaceonlinux#35]
```
What steps will reproduce the problem?
1. Breaks backspace in Windows (I know, its for linux)
2. if you use Chrome's feature that syncs your plugins between your various 
computers, its probably that this will get installed on your windows machine if 
you installed it on the linux one.
3.

What is the expected output? What do you see instead?
It should check if its running on windows and not do anything

What version of the product are you using? On what operating system?
0.52.0

Please provide any additional information below.


```

Original issue reported on code.google.com by `jasonble...@gmail.com` on 14 Sep 2013 at 11:00

**Comment by GoogleCodeExporter** [comment]
```
That should be a rather quick fix - I can try it out and submit a patch if that 
would help. 
I think the overhead of the extension running wouldn't be too big if we just 
don't attach the keydown listener if the OS is Windows.
It seems like this should be easy to check by using window.navigator.platform - 
I haven't tested what it outputs on Windows, but on Ubuntu 13.10 it says "Linux 
x86_64" - so I'm assuming it would be either "Windows..." or "Win...". 
```

Original comment by `nikolov....@gmail.com` on 26 Feb 2014 at 10:58

**Comment by noctarius** [comment]
Fixed in 1.0.1
